templates/referent/users_list.html.twig

```diff
@@ -259,10 +259,16 @@
                             <small>
                                 {% set is_host = user.isCommitteeHost %}
                                 {%- if user.isCommitteeSupervisor -%}
-                                    Animateur local
-                                    {{- is_host ? ' / ' -}}
-                                {%- endif -%}
-                                {{- is_host ? 'Co-animateur local' -}}
+                                    Animateur local {{- is_host ? ' / ' -}}
+                                {%- elseif is_host -%}
+                                    Co-animateur local
+                                {%- else -%}
+                                    Adhérent
+                                {% endif %}
+                            </small>
+                            <br />
+                            <small class="text--italic">
+                                {{ user.interests | join(', ') }}
```

Comment:
Not a big deal but can you just write `|join` instead of `| join` ? we don't add a whitespace for filters